src/05-features/byok/lib/__regression-sim__/T1-9-stdout.md

```
@@ -0,0 +1,197 @@
+
+변경 후 (무력화):
+
+```ts
+const iv = new Uint8Array(new ArrayBuffer(IV_LENGTH_BYTES));
+// 무력화 A: crypto.getRandomValues(iv); 를 주석 처리하여 IV를 모두 0으로 고정
+// crypto.getRandomValues(iv);
+```
+
+### Fail (예상 — 실측 FAIL 확인됨)
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)